src/components/MainLayout/MainPage/ScorePanel/PlayerCard/PlayerCardStyled.tsx

```diff
@@ -1,10 +1,12 @@
 import styled from 'styled-components';
 import { colors } from '../../../../../theme/variables';
+import { Type } from '../Player.types';
 
-export const PlayerCardStyled = styled.div`
+export const PlayerCardStyled = styled.div<{ type: Type }>`
   width: 100%;
-  height: 18%;
+  height: 100%;
   display: flex;
+  grid-area: ${(props) => (props.type === Type.player ? 'playerPlayerCard' : 'enemyPlayerCard')} ;
```

Comment:
why we have duplication in 'playerPlayerCard'?

Reply:
PlayerCard is a common component for both players. 'player' in 'playerPlayerCard' means a friendly card. For better understanding I can rename 'playerPlayerCard' to 'friendlyPlayerCard'

Reply:
sorry, understand... but i think it would be better to have other name like "gamerPlayerCard" or "ownPlayerCard"

Reply:
friendly is also ok